portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,57 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function showInfo(number) {
+  var elementId = "info-" + number;
```

Comment:
using `var` is general discouraged, because it addresses the global namespace. either `const` for immutable or `let` for mutable variables, both only exist in the local scope.

https://google.github.io/styleguide/jsguide.html#features-local-variable-declarations